health-sdk/sdk/src/main/java/net/gini/android/health/sdk/review/model/ExtractionResult.kt

```diff
@@ -2,9 +2,28 @@ package net.gini.android.health.sdk.review.model
 
 import net.gini.android.core.api.Resource
 
+/**
+ * Represents the result of processing a document to get its extractions.
+ */
+
+/**
+ * Wraps the result of the extraction request.
+ */
```

Comment:
[Request] Please combine the two comment blocks into one. Otherwise the first one will be ignore by kdoc.